build/source/elmfire_level_set.f90

```diff
@@ -981,6 +956,25 @@ SUBROUTINE LEVEL_SET_PROPAGATION(IWX_BAND,ICASE,NTIMESTEPS)
    ENDDO
 ENDIF
 
+INQUIRE(UNIT=712,OPENED=LOPEN)
```

Comment:
Shouldn't these next two lines be enclosed in the IF (DUMP_EMBER_FLUX_UMD) THEN statement immediately below?

Reply:
This is to close the file recording the embers generated in the Lagrangian scheme. File UNIT = 712 is written every timestep in the main while loop